lab-sam/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This looks great, but is a very basic package.json file, consider adding more information in future assignments.